ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -964,7 +964,11 @@ static PrjFS_Result HandleFileNotification(
     {
         // PrjFS_NotificationType_FileModified is a post-modified FileOp event (that cannot be stopped
         // by the provider) and so there's no need to check the result of the call to NotifyOperation
-        RemoveXAttr(fullPath, PrjFSFileXAttrName);
+        int error;
+        if (!RemoveXAttr(fullPath, PrjFSFileXAttrName, &error))
+        {
+            // TODO(Mac) #395: Log error (if the error is something other than ENOATTR).
```

Comment:
> Is there a valid reason why we'd try to remove an attribute that doesn't exist?

This could happen if two threads are processing `PrjFS_NotificationType_FileModified` for the same file at the same time.

They would each see that the `xattr` exists:

```
bool partialFile = GetXAttr(fullPath, PrjFSFileXAttrName, sizeof(PrjFSFileXAttrData), &xattrData);
```

And then one of them would win and remove the xattr (and the other wouldn't find it).

I'll put a few more details in the comment here.